modules/Core/assets/blockSounds/leaf.blocksounds

```diff
@@ -0,0 +1,5 @@
+{
+    "stepSounds": [],
+    "destroySounds": ["core:Leaves"],
```

Comment:
Should this (and the associated ogg) be named "leaf" instead of "leaves" ? I"m not sure what would be more consistent